### PR TITLE
Wrong DEFAULT_PATH. Should be pointing to BASE_DIR, not ROOT

### PR DESCRIPTION
**Pull request by gnalbandian**

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
Some tests seems to be failing following that change. Is using BASE_DIR a django convention ? If so we  could add a BASE_DIR = ROOT in the setting file.

Btw, are you using the reporting in latex ? I'm interested about your feedback.

**Comment by gnalbandian** [comment]
Hi @Pierre-Sassoulas. 

Sorry, haven't completely tested my suggested solution. But after some reading of your app, it's missing some parameters and structure changes to become a fully "pluggable app".
In the following link [https://docs.djangoproject.com/en/2.1/ref/applications/#django.apps.AppConfig](url) you can check how t obuild it properly.

Also you can check some other known django apps github repositories.

Basically what I saw is that you are missing is to subclass AppConfig where default app setting should be defined

**Comment by Pierre-Sassoulas** [comment]
Thank you, I'll look into it. I opened an issue regarding this (#21)

**Comment by gnalbandian** [comment]
Good job!
